### ISSUE TITLE
If you set Stretch amount to maximum (1024.0) - Render could offer "set to current stretch length" as a button

### ISSUE THREAD

**Issue by esaruoho** [essej/paulxstretch#10]
Expected: There's a way to render the "maximum length that you've set" by clicking on a button, instead of having to manually type in the number/length.

<img width="1085" alt="Näyttökuva 2022-4-30 kello 22 49 32" src="https://user-images.githubusercontent.com/4966687/166122194-df23b5b3-b139-4e43-8119-9ff02054067a.png">
<img width="524" alt="Näyttökuva 2022-4-30 kello 22 49 36" src="https://user-images.githubusercontent.com/4966687/166122196-4bad164d-2beb-4ead-bd55-95fb41e1dfc0.png">

**Comment by essej** [comment]
Good idea, it should probably just have the selected stretched length filled in there in hours, when it comes up. Having a button to reset it would be a bonus.
